AudioDeviceExample/AudioDevices/ExampleAVAudioEngineDevice.m

```diff
@@ -553,6 +586,12 @@ - (BOOL)startRendering:(nonnull TVIAudioDeviceContext)context {
 
 - (BOOL)stopRendering {
     @synchronized(self) {
+        
+        // Continue playing music even after disconnected from a Room.
+        if (self.continuousMusic) {
+            return YES;
```

Comment:
set self.renderingContext->deviceContext  to nil

Reply:
Are there any thread safety issues with clearing out the device context that we should be concerned about? The context is checked in the input nodes manual rendering block as well.

Reply:
Great point. Addressed the feedback.

Reply:
Look safe to me. I double checked the implementation and TVIExternalAudioDevice is passed as "this" for the context.